stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/runtime/ExecBallerina.java

```diff
@@ -0,0 +1,92 @@
+package org.ballerinalang.nativeimpl.runtime;
+
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.bre.bvm.BlockingNativeCallableUnit;
+import org.ballerinalang.model.types.TypeKind;
+import org.ballerinalang.model.values.BString;
+import org.ballerinalang.nativeimpl.io.channels.base.Buffer;
+import org.ballerinalang.natives.annotations.Argument;
+import org.ballerinalang.natives.annotations.BallerinaFunction;
+import org.ballerinalang.natives.annotations.ReturnType;
+import org.ballerinalang.util.exceptions.BallerinaException;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.Properties;
+
+/**
+ * Native function ballerina.runtime:exec.
+ *
+ * @since 0.970.0-alpha1
+ */
+@BallerinaFunction(
+        orgName = "ballerina", packageName = "runtime",
+        functionName = "execBallerina",
+        args = {@Argument(name = "command", type = TypeKind.STRING),
+                @Argument(name = "data", type = TypeKind.STRING)},
+        returnType = {@ReturnType(type = TypeKind.STRING)},
+        isPublic = true
+)
+public class ExecBallerina extends BlockingNativeCallableUnit {
+
+    @Override
+    public void execute(Context context) {
+
+        Properties property = System.getProperties();
+        String path = (String) property.get("ballerina.home");
+
+        String command = context.getStringArgument(0);
+        String packageName = context.getStringArgument(1);
```

Comment:
Validate to make sure package name does not contain only valid characters. Spaces ; | \ and anything that cannot be present in a ballerina package name should not be allowed here.